docs/docsite-index.md

```diff
@@ -65,8 +65,7 @@ An easy way to create beautiful apps with modular and customizable UI&nbsp;compo
     ```
     <!--{: .code-renderer.code-renderer--install }-->
 
-    If you are using Swift, don’t forget to uncomment the `use_frameworks!` line
-    at the top of your Podfile.
+    If you are using Swift, you may want to add `use_frameworks!` to your Podfile.
```

Comment:
FYI:
With CocoaPods 1.5.0, developers are no longer restricted into specifying use_frameworks! in their Podfile in order to install pods that use Swift. Interop with Objective-C should just work. However, if your Swift pod depends on an Objective-C, pod you will need to enable "modular headers" (see below) for that Objective-C pod.

Reply:
Cool. I'll just leave this out then.